native/src/main/java/io/ballerina/lib/aws/sqs/CreateQueueMapper.java

```diff
@@ -0,0 +1,88 @@
+    }
+
+    @SuppressWarnings("unchecked")
+    public static CreateQueueRequest getNativeCreateQueueRequest(BString queueName, BMap<BString, Object> bConfig) {
+        CreateQueueRequest.Builder builder = CreateQueueRequest.builder().queueName(queueName.getValue());
+
+        final Map<String, String> ATTRIBUTE_NAME_MAP = Map.ofEntries(
+                        Map.entry("delaySeconds", "DelaySeconds"),
+                        Map.entry("maximumMessageSize", "MaximumMessageSize"),
+                        Map.entry("messageRetentionPeriod", "MessageRetentionPeriod"),
+                        Map.entry("policy", "Policy"),
+                        Map.entry("receiveMessageWaitTimeSeconds", "ReceiveMessageWaitTimeSeconds"),
+                        Map.entry("visibilityTimeout", "VisibilityTimeout"),
+                        Map.entry("redrivePolicy", "RedrivePolicy"),
+                        Map.entry("redriveAllowPolicy", "RedriveAllowPolicy"),
+                        Map.entry("kmsMasterKeyId", "KmsMasterKeyId"),
+                        Map.entry("kmsDataKeyReusePeriodSeconds", "KmsDataKeyReusePeriodSeconds"),
+                        Map.entry("sqsManagedSseEnabled", "SqsManagedSseEnabled"),
+                        Map.entry("fifoQueue", "FifoQueue"),
+                        Map.entry("contentBasedDeduplication", "ContentBasedDeduplication"),
+                        Map.entry("deduplicationScope", "DeduplicationScope"),
+                        Map.entry("fifoThroughputLimit", "FifoThroughputLimit"));
+
+        if (bConfig != null) {
+            if (bConfig.containsKey(QUEUE_ATTRIBUTES)) {
+                BMap<BString, Object> attrs = (BMap<BString, Object>) bConfig.get(QUEUE_ATTRIBUTES);
+                if (attrs != null) {
+                    Map<QueueAttributeName, String> attrMap = new HashMap<>();
+                    for (Object key : attrs.getKeys()) {
+                        BString attrkey = (BString) key;
+                        Object value = attrs.get(attrkey);
+                        String awsAttrName = ATTRIBUTE_NAME_MAP.get(attrkey.getValue());
+                        if (awsAttrName != null && value != null) {
+                            attrMap.put(QueueAttributeName.fromValue(awsAttrName), value.toString());
+                        }
+                    }
+                    builder.attributes(attrMap);
+                }
+            }
+
+            if (bConfig.containsKey(TAGS)) {
+                BMap<BString, Object> tags = (BMap<BString, Object>) bConfig.get(TAGS);
+                if (tags != null) {
+                    Map<String, String> tagMap = new HashMap<>();
+                    for (Object key : tags.getKeys()) {
+                        BString tagKey = (BString) key;
+                        Object value = tags.get(tagKey);
+                        tagMap.put(tagKey.getValue(), value.toString());
+                    }
+                    builder.tags(tagMap);
+                }
```

Comment:
Don't need the casting here. Use the `getMapValue` method instead of plain `get` method. Check similar places.

```suggestion
                BMap<BString, Object> tags = bConfig.getMapValue(TAGS);
                if (tags != null) {
                    Map<String, String> tagMap = new HashMap<>();
                    for (Map.Entry<BString, Object> entrySet : tags.entrySet()) {
                        tagMap.put(entrySet.getKey().getValue(), entrySet.getValue().toString());
                    }
                    builder.tags(tagMap);
                }
```

Reply:
I tried implementing your recommended approach using `getMapValue()` instead of `get()`, but I encountered the below type mismatch error. 
`Type mismatch: cannot convert from BMap<capture#1-of ?,capture#2-of ?> to BMap<BString,Object> `
I fixed the above error using below approach. Is it okay?
``` 
 if (bConfig.containsKey(TAGS)) {
                var tags = bConfig.getMapValue(TAGS);
                if (tags != null) {
                    Map<String, String> tagMap = new HashMap<>();
                    for (var entrySet : tags.entrySet()) {
                        BString tagKey = (BString) entrySet.getKey();
                        Object value = entrySet.getValue();
                        tagMap.put(tagKey.getValue(), value.toString());
                    }
                    builder.tags(tagMap);
                }
            }
```

Reply:
Fixed with [58da165](https://github.com/ballerina-platform/module-ballerinax-aws.sqs/pull/90/commits/58da165b2ad78ca52b841c5e361bf6fe6eebd6e7)